non-wheel/active-jobs-dashboard/root.tpl

```diff
@@ -0,0 +1,64 @@
+<html>
+<head><title>Active Job Dashboard</title>
+<script type="text/javascript" src="jquery-latest.js"></script>
+<script type="text/javascript" src="jquery.tablesorter.min.js"></script>
```

Comment:
I'm thinking it might be easier for the user if we include jquery.tablesorter.min.js in this repo, and style.css too :)